manifest.json

```diff
@@ -43,11 +43,41 @@
         "128": "images/il_128.png"
     },
     "url_handlers": {
-        "openApp": {
+        "localhost/player": {
```

Comment:
The various IDs used make it easier to extract the pertinent information from the URL. It allows us to build a simple regex to remove the beginning of the URL and leave the relevant path and query string info in place.